### PR TITLE
MaXXII - Tracked Vehicle - Robot

### PR DESCRIPTION
**Pull request by robodyne-robotics**
I tried to use add_robot.py to add a new robot to the repository.

### CODE REVIEW

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: MaXXII - Rubber Tracked Robot
+date: 2022-06-19
+description:
+icon: /assets/img/robots/maxxii---rubber-tracked-robot/icon.jpg
+image: /assets/img/robots/maxxii---rubber-tracked-robot/image.jpg
+introduction: MaXXII is a ROS-ready skid-steering tracked mobile robotic platform which can be used for a large variety of applications as robot for agricolture and to speed up your robotics research!
+main-class: 'ground'
+color: '#FFFFFF'
+tags:
+- 'ROS-ready rubber tracked robot'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:12]:
```suggestion
```

This is a description not a tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- 'rubber tracked robot'
+- 'rubber tracked vehicle'
+- 'rubber tracks'
+- 'skid-steer'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:16]:
```suggestion
- 'skidsteer'
```

For consistency with existing tags.

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- 'skid-steering'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:17]:
```suggestion
```

Removing redundant tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- 'skid-steering robot'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:18]:
```suggestion
```

Removing redundant tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+- 'tank robot'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:19]:
```suggestion
```

Removing redundant tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- 'tank steering robot'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:20]:
```suggestion
```

Removing redundant tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+twitter_text: Checkout the MaXXII - Rubber Tracked Robot
+website: https://www.robo-dyne.com/maxxii-ugv-tracked-robot-vehicle/
+wiki_homepage:
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:23]: You appear to be missing a wiki page overview which is required with pointers to the developer documentation.

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: MaXXII - Rubber Tracked Robot
+date: 2022-06-19
+description:
+icon: /assets/img/robots/maxxii---rubber-tracked-robot/icon.jpg
+image: /assets/img/robots/maxxii---rubber-tracked-robot/image.jpg
+introduction: MaXXII is a ROS-ready skid-steering tracked mobile robotic platform which can be used for a large variety of applications as robot for agricolture and to speed up your robotics research!
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:8]:
```suggestion
introduction: MaXXII is a ROS-ready skid-steering tracked mobile robotic platform which can be used for a large variety of applications as robot for agriculture and to speed up your robotics research!
```
Spelling fixup

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+description:
+icon: /assets/img/robots/maxxii---rubber-tracked-robot/icon.jpg
+image: /assets/img/robots/maxxii---rubber-tracked-robot/image.jpg
+introduction: MaXXII is a ROS-ready skid-steering tracked mobile robotic platform which can be used for a large variety of applications as robot for agricolture and to speed up your robotics research!
+main-class: 'ground'
+color: '#FFFFFF'
+tags:
+- 'ROS-ready rubber tracked robot'
+- 'rubber tracked robot'
+- 'rubber tracked vehicle'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:14]:
```suggestion
```

Removing redundant tag

File: _posts/2022-06-19-maxxii---rubber-tracked-robot.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+date: 2022-06-19
+description:
+icon: /assets/img/robots/maxxii---rubber-tracked-robot/icon.jpg
+image: /assets/img/robots/maxxii---rubber-tracked-robot/image.jpg
+introduction: MaXXII is a ROS-ready skid-steering tracked mobile robotic platform which can be used for a large variety of applications as robot for agricolture and to speed up your robotics research!
+main-class: 'ground'
+color: '#FFFFFF'
+tags:
+- 'ROS-ready rubber tracked robot'
+- 'rubber tracked robot'
```
Comment (tfoote) [inline _posts/2022-06-19-maxxii---rubber-tracked-robot.md:13]:
```suggestion
```

Removing redundant tag

**Comment by tfoote** [comment]
@robodyne-robotics Do you have a moment to respond to the feedback?

**Comment by robodyne-robotics** [comment]
> @robodyne-robotics Do you have a moment to respond to the feedback?

Hello, yes, I will try to work on it during these days. Sorry, but I've been pretty busy so I couldn't solve it last week.

**Comment by tfoote** [comment]
Since this has been pending for a while I'm going to close it. Please feel free to open a new PR when you have the chance to update it.

**Pull request closed without merging**